src/components/shared/MarkerTooltipContents.js

```diff
@@ -24,6 +24,16 @@ function getMarkerDetails(data: MarkerPayload): React$Element<*> | null {
           </div>
         );
       }
+      case 'DOMEvent': {
+        return (
+          <div className="tooltipDetails">
+            <div className="tooltipLabel" key="type">
```

Comment:
TBH the key here is a leftover from a previous version of my patch, it's useless -- but also harmless.

Because I'd want see this patch landed rather now than later I don't ask you to remove it now, I'll file an issue for this.